### PR TITLE
fix: docs for s3

### PR DESCRIPTION
**Pull request by ypoplavs**
cleanup: deploy TK OSS in AWS
fix: use S3 as storage for TK Enterprise

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/articles/install/s3-storage.md [hunk 1]
```diff
@@ -4,29 +4,75 @@ To use S3 as storage, the steps are as follows:
...
+      "Action": [
+          "s3:GetObject",
+          "s3:PutObject"
+      ],
+      "Resource": "arn:aws:s3:::testkube-test-irsa/*"
+    }
+  ]
+}
+  ```  
+Replace `testkube-test-irsa` with the name of your own bucket.
```
Comment (dejanzele) [inline docs/articles/install/s3-storage.md:30]: Can we then remove this line and use `<BUCKET>` in the resource above?

File: docs/articles/install/s3-storage.md [hunk 2]
```diff
@@ -4,29 +4,75 @@ To use S3 as storage, the steps are as follows:
...
 
-
-2. Create a ServiceAccount with the ARN specified.
-   e.g.
+3. A Trust Relationship needs to be established in the IAM role to allow the EKS cluster to assume it:
```
Comment (dejanzele) [inline docs/articles/install/s3-storage.md:32]: > ``
```suggestion
3. A Trust Relationship needs to be established in the IAM role to allow the Testkube API's ServiceAccount to assume it:
```

File: docs/articles/deploying-in-aws.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Please mind that is it necessary to install [EBS CSI driver](https://docs.aws.am
 
 ## Ingress and Service Resources Configuration
 
-To deploy and expose Testkube to the outside world, you will need to create an ingress for Testkube's API server. In this tutorial, we will be updating `values.yaml` that later will be passed to the `helm install` command.
+To deploy and expose Testkube API to the outside world, you will need to create an ingress for Testkube's API server. In this tutorial, we will be updating `values.yaml` that later will be passed to the `helm install` command.
```
Comment (dejanzele) [inline docs/articles/deploying-in-aws.md:23]:
```suggestion
To deploy and expose Testkube API to the outside world, you will need to create an Ingress resource for Testkube's API server. In this tutorial, we will be updating `values.yaml` that later will be passed to the `helm install` command.
```

**Review by emil2k — APPROVED** [review]

File: docs/articles/deploying-in-aws.md [hunk 4]
```diff
@@ -66,69 +66,12 @@ helm install --create-namespace testkube kubeshop/testkube --namespace testkube
 
 After the installation command is complete, you will see the following resources created into your AWS Console.
 
-![AWS Console](../img/aws-resource-console.png)
-
 ![AWS Console2](../img/aws-resource-console-2.png)
 
-![AWS Console3](../img/aws-resource-console-3.png)
-
 Please note that the annotations may vary, depending on your Load Balancer schema type, backend-protocols (you may use http only), target-type, etc. However, this is the bare minimum that should be applied to your configuration.
```
Comment (dejanzele) [inline docs/articles/deploying-in-aws.md:71]: This part `(you may use http only)` might be a bit misleading, does it mean if you want to use http or you can use http only?

Comment (ypoplavs) [reply docs/articles/deploying-in-aws.md]: badly phrazed

**Pull request merged**